.github/workflows/ci_compliance.yml

```diff
@@ -29,3 +33,31 @@ jobs:
           aws-s3-bucket-name: s2n-tls-ci-artifacts
           aws-s3-region: us-west-2
           cdn: https://d3fqnyekunr9xg.cloudfront.net
+
+      # In order to accurately check if there are any uncommitted changes, cleanup any
+      # intermediate artifacts generated during the duvet run.
+      - name: Cleanup intermediate artifacts
+        run: rm -r specs
+        shell: bash
+
+      - name: Extract RFC spec data
+        working-directory: ./compliance
+        run: ./initialize_duvet.sh
+        shell: bash
+
+      - name: Check if there are uncommitted changes
+        run: |
+          # If this fails you need to run `cd compliance && ./compliance/initialize_duvet.sh`
+          #
+          # Use 'git status --porcelain' instead of 'git diff --exit-code' since git diff
+          # only detects diffs but fails to detect new files. Ignore the s2n-quic dir
+          # `(:!s2n-quic)` since we explicitly clone the repo as part of this job.
+          git_status=$(git status --porcelain -- ':!s2n-quic')
+          if [ -n "$git_status" ]; then
+            echo "Found uncommitted changes:"
+            echo "$git_status"
```

Comment:
What's up with the newlines in your output? Is this from an earlier version of this script?
```
Found uncommitted changes \n?? compliance/specs/tools.ietf.org/rfc/rfc8422.txt
?? compliance/specs/tools.ietf.org/rfc/rfc8422/
```

Reply:
yep its an earlier version. the newline didnt actually render a newline so i broke it into two `echo` statements instead